docs/DOCKER.md

```diff
@@ -0,0 +1,17 @@
+# Docker files details
+
+## Dockerfile
+
+    The purpose of this file is to facilitate the setup of Data Management Service(DMS) docker image.
+    It utilizes the assets and dlls from "src\frontend\EdFi.DataManagementService.Frontend.AspNetCore", src\backend\EdFi.DataManagementService.Backend.Installer\EdFi.DataManagementService.Backend.Installer
+    folders.
+
+### Environment variables details for running the container:
```

Comment:
Should this be `##` instead of `EEE`? No need for a colon at the end of a header line.